admin_guide/cluster_management.rst

```diff
@@ -145,3 +145,24 @@ The new shard(s) are created on the same node as the shard(s) from which the ten
     102240,
     'source_host', source_port,
     'dest_host', dest_port);
+
+Database Bloat
```

Comment:
Instead of 'Database Bloat' as a title, could we use something like just "Vacuuming Distributed Tables"?